homeassistant/components/sma/sensor.py

```diff
@@ -187,42 +143,46 @@ def name(self):
     @property
     def state(self):
         """Return the state of the sensor."""
-        return self._state
+        return self._sensor.value
 
     @property
     def unit_of_measurement(self):
         """Return the unit the value is expressed in."""
         return self._sensor.unit
 
-    @property
-    def extra_state_attributes(self):  # Can be remove from 0.99
-        """Return the state attributes of the sensor."""
-        return self._attr
-
     @property
     def poll(self):
```

Comment:
```suggestion
    def should_poll(self):
```

Reply:
But you can just delete it because its provided by `CoordinatorEntity`

Reply:
Removed in 2317fbb